notifications/src/main/java/com/dojo/notifications/grpc/leaderboard/LeaderboardClient.java

```diff
@@ -95,16 +90,15 @@ public void onCompleted() {
         });
     }
 
-    private TreeSet<Participant> getParticipants(LeaderboardResponse leaderboardResponse) {
+    private Leaderboard getLeaderboard(LeaderboardResponse leaderboardResponse) {
         TreeSet<Participant> participants = new TreeSet<>();
```

Comment:
we usually set the collection type to be the interface, e.g 'Set', for better abstraction and use of polymorphism